src/main/java/orm/QueryRenderer.java

```diff
@@ -0,0 +1,49 @@
+package orm;
+
+import orm.dsl.condition.Condition;
+import orm.util.CollectionUtils;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class QueryRenderer {
+
+    /**
+     * 컬럼들을 콤마로 구분하여 반환
+     */
+    public String joinColumnNamesWithComma(List<? extends TableField> columns) {
+        return columns.stream()
+                .map(TableField::getFieldName)
+                .collect(Collectors.joining(","));
+    }
+
+    /**
+     * 대량 삽입 SQL 문에 대한 값의 문자열 표현을 렌더링 합니다.
+     *
+     * <p>예를 들어, 이러한 형태가 됩니다 : (id,name,age) </p>
+     **/
+    public String renderBulkInsertValues(List<List<? extends TableField>> columns) {
+        List<String> result = new ArrayList<>(columns.size());
+        for (List<? extends TableField> inertValue : columns) {
+            result.add("(%s)".formatted(inertValue.stream()
+                    .map(TableField::getFieldValue)
+                    .map(String::valueOf)
+                    .collect(Collectors.joining(",")))
+            );
+        }
+
+        return String.join(", ", result);
+    }
+
+    public String renderWhere(List<Condition> conditions) {
+        if (CollectionUtils.isEmpty(conditions)) {
+            return "";
+        }
+
+        return " WHERE " +
+                conditions.stream()
+                        .map(Condition::toString)
+                        .collect(Collectors.joining(" AND "));
+    }
```

Comment:
```suggestion
    public String renderWhere(Conditions conditions) {
        if (conditions.isEmpty()) {
            return "";
        }

        return " WHERE " + conditions
                .join(" AND ")
                .toQuery();
```

반영을 위한 코멘트는 아닙니다. 
저는 `List<Condition>`상태를 관리하는 객체가 있으면 어떨까 싶어 공유해봅니다~

Reply:
저도 생각해보니 Condition을 일급객체로 관리하면 훨씬 좋다고 생각되네요 감사합니다!